_logs/bias-3e010137-10fc-46f7-a68b-7649ff13cc7d.txt

```diff
@@ -0,0 +1,2201 @@
+|   3  NVIDIA H100 PCIe               On  | 00000000:00:0A.0 Off |                    0 |
+| N/A   38C    P0              79W / 350W |    963MiB / 81559MiB |      0%      Default |
+|                                         |                      |             Disabled |
++-----------------------------------------+----------------------+----------------------+
+|   4  NVIDIA H100 PCIe               On  | 00000000:00:0B.0 Off |                    0 |
+| N/A   37C    P0              77W / 350W |    963MiB / 81559MiB |      0%      Default |
+|                                         |                      |             Disabled |
++-----------------------------------------+----------------------+----------------------+
+|   5  NVIDIA H100 PCIe               On  | 00000000:00:0C.0 Off |                    0 |
+| N/A   36C    P0              77W / 350W |    963MiB / 81559MiB |      0%      Default |
+|                                         |                      |             Disabled |
++-----------------------------------------+----------------------+----------------------+
+|   6  NVIDIA H100 PCIe               On  | 00000000:00:0D.0 Off |                    0 |
+| N/A   43C    P0              83W / 350W |    963MiB / 81559MiB |      0%      Default |
+|                                         |                      |             Disabled |
++-----------------------------------------+----------------------+----------------------+
+|   7  NVIDIA H100 PCIe               On  | 00000000:00:0E.0 Off |                    0 |
+| N/A   38C    P0              78W / 350W |    963MiB / 81559MiB |      0%      Default |
+|                                         |                      |             Disabled |
++-----------------------------------------+----------------------+----------------------+
+                                                                                         
++---------------------------------------------------------------------------------------+
+| Processes:                                                                            |
+|  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
+|        ID   ID                                                             Usage      |
+|=======================================================================================|
++---------------------------------------------------------------------------------------+
+
+====================================================================================================
+Training DataLoader: total number of tokens: 1000000000 across 10 files
+Validation DataLoader: total number of tokens: 100000000 across 1 files
+====================================================================================================
+Centered output layer, initial loss -10.826 => -7.657 (0.1s)
+step:0/1480 val_loss:7.7109 train_time:0ms step_avg:nanms
```

Comment:
This is the basic trick: Setting output layer allows training to start @ `7.7` instead of 10.8`